WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,55 @@
+
+## Daily routine
+
+1. [Check code review requests](https://github.com/zkSNACKs/WalletWasabi/pulls). __Reviews / Awaiting review from you__.
+2. Check [priority issues and PR-s!](https://github.com/zkSNACKs/WalletWasabi/labels/priority). Take it or set Assignees and Labels if you can.
+3. Check your [own PR-s and answer](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+repo%3AzkSNACKs%2FWalletWasabi+) if necessary. __Click Created__.
+   - If you need help use __@mention__.
+   - If you need review __set Reviewers__.
+4. Check [assigned issues](https://github.com/zkSNACKs/WalletWasabi/issues) __Click Assignee / Select your profile__.
+5. Check [mentions](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+repo%3AzkSNACKs%2FWalletWasabi+) __Click Mentioned__.
+6. Check the bell.
+7. Work on anything you want - assign it to yourself.
+
+## Weekly routine
+
+- Wednesday: Developer meeting. Use [dev meeting board](https://github.com/orgs/zkSNACKs/projects/1) to manage tasks.
+- Pair programming hour.
+
+## Monthly routine
+
+- Go through all Issues and Pull requests and triage them.
+
+## Roadmap
+
+https://github.com/zkSNACKs/WalletWasabi/projects/8
+
+## Code review (CR)
+
+- Read the description of the Pull Request. If it is missing ask for it if necessary.
+- Check Continuous Integration (CI) before code review. Only continue if it is passed.
+- Do not review code longer than 60 minutes or more than 400 lines. Take a 10 minute pause at that point.
+- Check for code smells. Give constructive feedback that helps (Not hurts).
+  - Easy to read.
+  - Maintainability.
+- Checkout PR and test it if required.
+- Request changes if needed.
+- Approve if you are done.
+
+__Tips__: if the PR is too big ask for breakdown.
+
+__Fact__: code review is the top rated code improvement tool.
```

Comment:
```suggestion
__Fact__: code review is one of the few scientifically proven code improvement tool.
```

Reply:
If Nopara's suggestion is accepted it should be `tools` maybe not `tool`